### ISSUE TITLE
Consider changing tax rate inputs for improved accessibility

### ISSUE THREAD

**Issue by robincornett** [easydigitaldownloads/easy-digital-downloads#7730]
### Explain your enhancement (please be detailed)
Tax rates should not be able to be adjusted incrementally, so this setting input may be better as a text input.

- source: [Why the GOV.UK Design System team changed the input type for numbers](https://technology.blog.gov.uk/2020/02/24/why-the-gov-uk-design-system-team-changed-the-input-type-for-numbers/)
- source: HTML Standard ([When number is not appropriate](https://html.spec.whatwg.org/multipage/input.html#when-number-is-not-appropriate))

### Justification or use case
Other instances of `input type="number"` in EDD are quantity related, so they can be increased/decreased incrementally, but tax rates should be entered specifically, and not accidentally adjusted by an errant mouse scroll.

**Comment by spencerfinnell** [comment]
@robincornett To confirm, the `number` field isn't currently preventing valid inputs, correct? The change would be to stop things like this from happening:

![2020-04-14 09 44 24](https://user-images.githubusercontent.com/491124/79231792-9b78b600-7e34-11ea-8b01-97a0b0b617fc.gif)

We would need to add some client-side validation if this change is made.

**Comment by robincornett** [comment]
Yes, since the tax rates are added using JS--there is an error message, similar to the HTML5 messages, if it's a standard PHP submission.
